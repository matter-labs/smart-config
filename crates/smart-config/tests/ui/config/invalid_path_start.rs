fn main() {
    smart_config::config!("!what": 1);
}
