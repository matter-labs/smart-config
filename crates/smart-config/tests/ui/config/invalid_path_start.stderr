error[E0080]: evaluation of constant value failed
 --> tests/ui/config/invalid_path_start.rs:2:5
  |
2 |     smart_config::config!("!what": 1);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the evaluated program panicked at 'Path #0 `!what` is invalid: name contains a disallowed char '!' at position 0; allowed chars are [_a-z]', $DIR/tests/ui/config/invalid_path_start.rs:2:5
  |
  = note: this error originates in the macro `smart_config::config` (in Nightly builds, run with -Z macro-backtrace for more info)
